packages/koa-shopify-graphql-proxy/src/shopify-graphql-proxy.ts

```diff
@@ -2,24 +2,36 @@ import proxy from 'koa-better-http-proxy';
 import {Context} from 'koa';
 
 export const PROXY_BASE_PATH = '/graphql';
-export const GRAPHQL_PATH = '/admin/api/graphql.json';
+export const GRAPHQL_PATH_PREFIX = '/admin/api';
 
-export interface ProxyOptions {
+export enum ApiVersion {
+  '2019-04' = '2019-04',
+  '2019-09' = '2019-09',
```

Comment:
Can we use nicer names for these? `April19`/ `September19` maybe?

Reply:
You make me realize the API version is not September but July. So I did the changes you propose 👍